lib/utils/bootstrap-ui-package.ts

```diff
@@ -89,5 +89,8 @@ export function bootstrapUIPackage(
   context.log.info(`Building library for development`);
   spawnSyncStrict('npm', ['run', 'build:lib'], defaultSpawnOptions);
 
+  context.log.info('Running ngcc to avoid build errors');
+  spawnSyncStrict('npm', ['run', 'ngcc'], defaultSpawnOptions);
```

Comment:
This is only one the initial project bootstrap, right?

I get out of memory/call stack errors if I run `ngcc` directly on existing projects because it tries to bundle dependencies in multiple module formats.

Reply:
Yes, this is only for the initial project bootstrap. Some dependency made `npm start` stop working after a `lsc create app`. To workaround that it needs to run `ngcc` before the initial start